src/rgw/rgw_sync_log_trim.cc

```diff
@@ -737,6 +737,25 @@ class RecentEventList {
 
 namespace rgw {
 
+// read bucket trim configuration from ceph context
+void configure_bucket_trim(CephContext *cct, BucketTrimConfig& config)
+{
+  auto conf = cct->_conf;
+
+  config.trim_interval_sec =
+      conf->get_val<int64_t>("rgw_sync_log_trim_interval");
+  config.counter_size = 512;
+  config.buckets_per_interval =
+      conf->get_val<int64_t>("rgw_sync_log_trim_max_buckets");
+  config.min_cold_buckets_per_interval =
+      conf->get_val<int64_t>("rgw_sync_log_trim_min_cold_buckets");
+  config.concurrent_buckets =
+      conf->get_val<int64_t>("rgw_sync_log_trim_concurrent_buckets");
+  config.notify_timeout_ms = 10;
```

Comment:
10ms for notify timeout? I'd go with 2 seconds

Reply:
oops, i intended to use 10 sec here. that `_ms` suffix was supposed to prevent that kind of mistake :)

Reply:
yeah 10 sec is probably better.